cdap-app-fabric/src/main/java/io/cdap/cdap/app/preview/PreviewRequestQueueState.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.app.preview;
+
+import com.google.common.base.Objects;
+
+/**
+ * Class representing the state of the preview job queue.
+ */
+public class PreviewRequestQueueState {
+  private long numOfPreviewWaiting;
```

Comment:
final

Reply:
Also, call it `pendingRequests`

Reply:
BTW, it can be just an `int` instead of `long`, right?

Reply:
fixed.